apps/server/src/services/TimerService.ts

```diff
@@ -77,8 +85,9 @@ export class TimerService {
   /**
    * Reloads information for currently running timer
    * @param timer
+   * @param initialData
    */
-  hotReload(timer) {
+  hotReload(timer, initialData?: initialLoadingData, playback?: Playback, addedTime?: number, pausedAt?: number) {
```

Comment:
I dont think we should re-use the hot reload, that is not the scope of this function.

We would likely prefer adding a method `init()` to the TimerService to allow it to be re-initialised after the constructer being called